Lib/fontTools/ttLib/ttFont.py

```diff
@@ -491,6 +491,13 @@ def getGlyphOrder(self):
 				# in combination with the Adobe Glyph List (AGL).
 				#
 				self._getGlyphNamesFromCmap()
+			elif len(glyphOrder) < self['maxp'].numGlyphs:
```

Comment:
you could also merget this `elif` branch with the above `if` by doing `if glyphOrder is None or len(glyphOrder) < self.["maxp"].numGlyphs:` (while also moving the accompanying comments), since both are doing the same (i.e. calling `_getGlyphNamesFromCmap`)

Reply:
or maybe actually, you should keep the `elif` but add a logging.warning to this one, since we are basically discarding the information in the post format1 table (first 258 glyphs named using standard mac order) and treating it as if it were format 3 (no names at all).

Reply:
Rebased on `main` and added warning log. PTAL

Reply:
Interestingly, the CI changes the logging level
https://github.com/fonttools/fonttools/runs/8096754180?check_suite_focus=true#step:5:387
What's the advice? Remove the `levelname` assert?

Reply:
Nevermind, figured out what was going on. There's another test that sets the logging level.